article/_posts/2022-01-14-CAVAT.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: review
+title: "Context-aware virtual adversarial training for anatomically-plausible segmentation"
+tags: adversarial
+author: "Thierry Judge"
+cite:
+    authors: "Ping Wang, Jizong Peng, Marco Pedersoli, Yuanfeng Zhou, Caiming Zhang, Christian Desrosiers"
+    title:   "Context-aware virtual adversarial training for anatomically-plausible segmentation"
+    venue:   "MICCAI 2019"
+pdf: "https://arxiv.org/pdf/2107.05532.pdf"
+---
+
+# Introduction 
+
+Authors tackle the task of generating anatomically plausible segmentations using an adversarial training scheme while 
+optimizing a non-differentiable loss function with the REINFORCE algorithm.
+
+Authors start by listing a number of methods that attempt to add constraints to neural network training such as size 
+constraints and centroid positions. These constraints require differentiable loss functions which limit their use. 
+Authors would like to exploit the information contained in anatomical shape priors like in Oktay et al., 2017 (ACNN), Zotti
+et al., 2018 (Prior-aware segmentation) and Painchaud et al. 2020. 
+
+Authors explore the aspect of semi-supervised learning as training segmentation networks to learn valid shapes is a 
+difficult task when little labeled data is available. Authors, therefore, propose a method base on Virtual Adversarial 
+Training (VAT) and REINFORCE to optimize a non-differentiable local connectivity constraint. 
+
+# Method 
+
+## Note on VAT 
+The goal of the VAT paper (Virtual Adversarial Training: A Regularization Method for Supervised and Semi-Supervised 
```

Comment:
If the paper is so relevant, adding the reference would be useful I think.

Reply:
Fixed in bdcc473